Header header
float32 rotx
float32 roty
float32 rotz

float32 posx
float32 posy
float32 posz
